analyzer-models/src/test/java/ffb/analyzer/models/espn/TeamTests.java

```diff
@@ -0,0 +1,82 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import ffb.analyzer.models.espn.Team;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.net.URL;
+
+public class TeamTests {
+    private static final String TEAMS = "teams.json";
+    private static final int EXPECTED_TEAM_COUNT = 10;
+
+    private static ObjectMapper MAPPER;
+
+    @BeforeClass
+    public static void buildPrep() {
+        MAPPER  = new ObjectMapper().enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
+        MAPPER.disable(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES);
+    }
+
+    @Test
+    public void testTeamSerialization() throws JsonProcessingException, MalformedURLException {
+        DraftStrategy strategy = new DraftStrategy();
+
+        List<String> owners = new ArrayList<String>();
```

Comment:
If you want to save a few key strokes -- `List.of(...)` is helpful when you know what the list should be populated with. Has the added benefit of being immutable as well.

Reply:
I'm a fan of shortcuts. Updated.